api/api.js

```diff
@@ -2567,3 +2574,39 @@ const getPublishedThroughHubSubQuery = (query) => {
 
   return publishedThroughHubSubQuery
 }
+
+const processReleaseCollectedTransaction = async (txId) => {
+  try {
+    await NinaProcessor.init();
+    const tx = await NinaProcessor.provider.connection.getParsedTransaction(txId, {
+      commitment: 'confirmed',
+      maxSupportedTransactionVersion: 0
+    })
+    if (tx) {
+      let accounts = tx.transaction.message.instructions.find(i => i.programId.toBase58() === process.env.NINA_PROGRAM_ID)?.accounts
+      if (accounts && tx.meta.logMessages.some(log => log.includes('ReleasePurchase'))) {
+        let releasePublicKey = accounts[2].toBase58()
+        let accountPublicKey = accounts[1].toBase58()
+        await NinaProcessor.addCollectorForRelease(releasePublicKey, accountPublicKey)
+      } else if (accounts && tx.meta.logMessages.some(log => log.includes('ReleaseClaim'))) {
+        let releasePublicKey = accounts[1].toBase58()
+        let accountPublicKey = accounts[3].toBase58()
+        await NinaProcessor.addCollectorForRelease(releasePublicKey, accountPublicKey)
+      } else if (!accounts || accounts.length === 0) {
+        for (let innerInstruction of tx.meta.innerInstructions) {
+          for (let instruction of innerInstruction.instructions) {
+            if (instruction.programId.toBase58() === 'ninaN2tm9vUkxoanvGcNApEeWiidLMM2TdBX8HoJuL4') {
```

Comment:
should use a constant variable for the program id that can be used across the api code.